LetsDinnerV2 MessagesExtension/Controller/RegistrationVC/RegistrationViewController.xib

```diff
@@ -0,0 +1,389 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I deleted this xib, why is it still here?

Reply:
thats is the merge conflicts. I wanna delete the old file, but i cannot find the way yet